awswrangler/dynamodb/_read.py

```diff
@@ -272,9 +196,8 @@ def _read_items(
     return items
 
 
-def read_items(  # pylint: disable=too-many-branches
+def read_items(
     table_name: str,
-    index_name: Optional[str] = None,
```

Comment:
This change was made 4 hours ago, I don't think we need to remove it :)
https://github.com/aws/aws-sdk-pandas/pull/1961